etcdserver/corrupt.go

```diff
@@ -264,43 +271,29 @@ func (s *EtcdServer) getPeerHashKVs(rev int64) (resps []*peerHashKVResp) {
 		if len(p.eps) == 0 {
 			continue
 		}
-		cli, cerr := clientv3.New(clientv3.Config{
-			DialTimeout: s.Cfg.ReqTimeout(),
-			Endpoints:   p.eps,
-		})
-		if cerr != nil {
-			lg.Warn(
-				"failed to create client to peer URL",
-				zap.String("local-member-id", s.ID().String()),
-				zap.String("remote-peer-id", p.id.String()),
-				zap.Strings("remote-peer-endpoints", p.eps),
-				zap.Error(cerr),
-			)
-			continue
-		}
 
 		respsLen := len(resps)
-		for _, c := range cli.Endpoints() {
+		var lastErr error
+		for _, ep := range p.eps {
```

Comment:
Can we rename the `pps` `p` `eps`? It is not easy to understand and remember. I think the naming way of `getVersion()` and `getVersions()` in [cluster_util.go](https://github.com/etcd-io/etcd/blob/master/etcdserver/cluster_util.go#L140) is much clear.

Reply:
Thanks! I tried to make it clearer. I did keep eps as short for endpoints.